packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md

```diff
@@ -14,13 +14,25 @@ Are you an admin? See the Identity Engine [Limitations](https://help.okta.com/ok
 
 ### Classic Engine features not supported in the Identity Engine
 
+#### Custom sign-in page for embedded app links
+
+**What Changed:** Using a custom sign-in page for embedded app links isn't supported. Users who click an app embed link are now evaluated by their org's Okta sign-in policy. Admins can customize an Okta-hosted sign-in page or configure an IdP routing rule for the app.
+
+**Further information:** [Configure a custom Okta-hosted sign-in page](/docs/guides/style-the-widget/before-you-begin/) and [Configure routing rules](https://help.okta.com/okta_help.htm?type=oie&id=ext_Identity_Provider_Discovery)
+
+***
+
 #### Event Type availability for Event Hooks
 
 **What Changed:** The following Event Types aren't available in the Identity Engine because Device Trust isn't currently supported:
 
 * `user.authentication.authenticate`
 * `user.credential.enroll`
 
```

Comment:
@rameshbhojan-okta should we include the obsolete EventTypeV1 events also here?
CORE_USER_AUTH_INVALID_CERTIFICATE
CORE_USER_AUTH_LOGIN_FAILED
CORE_USER_AUTH_LOGIN_SUCCESS
https://docs.google.com/spreadsheets/d/18pu08393Nafnry1jrq410RLKtv0VEAlrA4iMyOYn75k/edit#gid=358669831